app/views/filters/_form.html.erb

```diff
@@ -1,14 +1,12 @@
-<% javascript 'filters' %>
-
 <%= form_for @filter do |f| %>
   <%= base_errors_for f.object %>
   <ul class="nav nav-tabs" data-tabs="tabs">
     <li class="active"><a href="#primary" data-toggle="tab"><%= _("Filter") %></a></li>
     <% if show_location_tab? %>
-      <li><a href="#locations" data-toggle="tab"><%= _("Locations") %></a></li>
+      <li><a href="#locations" data-toggle="tab" style=<%= display(!@filter.override?) %>><%= _("Locations") %></a></li>
```

Comment:
This is missing the `allow-locations` condition that was checked before, some resources have override but only allow overriding one of the taxonomies.